components/match2/wikis/splitgate/match_group_input_custom.lua

```diff
@@ -468,18 +467,6 @@ function mapFunctions.getScoresAndWinner(map)
 	return map
 end
 
-function mapFunctions.getTournamentVars(map)
```

Comment:
this is a functional change
or was the splitgate infobox league added by now?

Reply:
map will inherit match vars that are set in [matchFunctions.getTournamentVars](https://github.com/Liquipedia/Lua-Modules/blob/28b181b12039ec4de5d78a4594c67dd833e827a9/components/match2/wikis/splitgate/match_group_input_custom.lua#L289C10-L289C24)